### PR TITLE
🌐 Add Spanish translation for `docs/es/docs/benchmarks.md`

### PR DESCRIPTION
**Pull request by pablocm83**
- Version >= 0.108.0

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 0c8301b51962a7292cd16199efbab68ddf9c9e78 at: https://78dc7aa2.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 47030998ebe7e38d0fc67a2b1edd314ee1263ceb at: https://ac9ac6e0.fastapitiangolo.pages.dev

**Review by alejsdev — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 3f2d2f29ac71187fada85e3c8eedd7f48617234f at: https://4b342c6b.fastapitiangolo.pages.dev

File: docs/es/docs/benchmarks.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+Específicamente, para ver Uvicorn, Starlette y FastAPI comparadas entre sí (entre muchas otras herramientas).
+
+Cuanto más sencillo sea el problema resuelto por la herramienta, mejor rendimiento obtendrá. Y la mayoría de los benchmarks no prueban las funciones adicionales proporcionadas por la herramienta.
+
+La jerarquía sería:
+
+* **Uvicorn**: como servidor ASGI
+    * **Starlette**: (usa Uvicorn) un microframework web
+         * **FastAPI**: (usa Starlette) un microframework API con varias características adicionales para construir API, con validación de datos, etc.
```
Comment (tiangolo) [inline docs/es/docs/benchmarks.md:19]:
```suggestion
         * **FastAPI**: (usa Starlette) un microframework API con varias características adicionales para construir APIs, con validación de datos, etc.
```

File: docs/es/docs/benchmarks.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    * Si estás comparando Starlette, compáralo con Sanic, Flask, Django, etc. Frameworks web (o microframeworks).
+* **FastAPI**:
+    * De la misma manera que Starlette usa Uvicorn y no puede ser más rápido que él, **FastAPI** usa Starlette, por lo que no puede ser más rápido que él.
+    * * FastAPI ofrece más características además de las de Starlette. Funciones que casi siempre necesitas al crear una API, como validación y serialización de datos. Y al usarlo, obtienes documentación automática de forma gratuita (la documentación automática ni siquiera agrega gastos generales a las aplicaciones en ejecución, se genera al iniciar).
+    * Si no usó FastAPI y usó Starlette directamente (u otra herramienta, como Sanic, Flask, Responder, etc.), tendrías que implementar toda la validación y serialización de datos tu mismo. Por lo tanto, tu aplicación final seguirá teniendo la misma sobrecarga que si se hubiera creado con FastAPI. Y en muchos casos, esta validación y serialización de datos constituye la mayor cantidad de código escrito en las aplicaciones.
```
Comment (tiangolo) [inline docs/es/docs/benchmarks.md:31]:
```suggestion
    * Si no usaras FastAPI y usaras Starlette directamente (u otra herramienta, como Sanic, Flask, Responder, etc.), tendrías que implementar toda la validación y serialización de datos tu mismo. Por lo tanto, tu aplicación final seguirá teniendo la misma sobrecarga que si se hubiera creado con FastAPI. Y en muchos casos, esta validación y serialización de datos constituye la mayor cantidad de código escrito en las aplicaciones.
```

File: docs/es/docs/benchmarks.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    * Entonces, al usar FastAPI estás ahorrando tiempo de desarrollo, errores, líneas de código y probablemente obtendrías el mismo rendimiento (o mejor) que obtendrías si no lo usaras (ya que tendrías que implementarlo todo en tu código).
+    * Si estás comparando FastAPI, compáralo con un framework de aplicación web (o conjunto de herramientas) que proporciona validación, serialización y documentación de datos, como Flask-apispec, NestJS, Molten, etc. Frameworks con validación, serialización y documentación automáticas integradas.
```
Comment (tiangolo) [inline docs/es/docs/benchmarks.md:33]:
```suggestion
    * Si estás comparando FastAPI, compáralo con un framework de aplicaciones web (o conjunto de herramientas) que proporciona validación, serialización y documentación de datos, como Flask-apispec, NestJS, Molten, etc. Frameworks con validación, serialización y documentación automáticas integradas.
```

**Comment by tiangolo** [comment]
Thanks @pablocm83! 🍰

We added just a few tweaks and now it's done. 🤓 

Thanks @alejsdev for the help! 🍪

**Pull request merged**